src/tokens/TokenBalanceItem.tsx

```diff
@@ -0,0 +1,129 @@
+import BigNumber from 'bignumber.js'
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+import { StyleSheet, Text, View } from 'react-native'
+import { AssetsEvents } from 'src/analytics/Events'
+import ValoraAnalytics from 'src/analytics/ValoraAnalytics'
+import { formatValueToDisplay } from 'src/components/TokenDisplay'
+import TokenIcon from 'src/components/TokenIcon'
+import Touchable from 'src/components/Touchable'
+import { getLocalCurrencySymbol, usdToLocalCurrencyRateSelector } from 'src/localCurrency/selectors'
+import useSelector from 'src/redux/useSelector'
+import colors from 'src/styles/colors'
+import { typeScale } from 'src/styles/fonts'
+import { Spacing } from 'src/styles/styles'
+import { TokenBalance } from 'src/tokens/slice'
+
+const TokenBalanceInLocalCurrency = (token: TokenBalance) => {
```

Comment:
I think this should be merged with the refactored TokenDisplay component on #4234 which supports tokenIds. Could be in a follow up after that is merged.